ai.menlo.jan.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Jan
+Comment=Local AI Assistant that runs 100% offline
+Exec=jan
+Icon=ai.menlo.jan
+Type=Application
+Categories=Development;AI;
```

Comment:
`AI` is an invalid category

See https://specifications.freedesktop.org/menu-spec/latest/category-registry.html